src/index.html

```diff
@@ -15,8 +15,103 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <!-- <link rel="stylesheet" href="https://fonts.google.com/specimen/Roboto"> -->
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;
+      0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
```

Comment:
The `href` attribute in the `link` tag for Google Fonts is too long. It should be split into multiple lines to adhere to the 80 character limit per line. You could place each font weight on a new line to improve readability.